docker-compose.test.yml

```diff
@@ -16,7 +16,7 @@ services:
 
   sut:
     build: .
-    command: sh entrypoint.sh
+    command: "bash -c 'python3 src/manage.py migrate && python3 src/manage.py loaddata init_data.json && python3 src/manage.py test'"
```

Comment:
I think you want a [MakeFile](https://www.gnu.org/software/make/manual/make.html) for this.

It gives you access to `Package.json`-esqe commands.
Nothing that needs to be implemented now. But to keep in mind for the future